src/comp/comp.py

```diff
@@ -21,51 +21,53 @@ def __repr__(self):
     Human("David", 31),
 ]
 
+print(humans[0].name)
+
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [human.name for human in humans if human.name[0] == 'D']
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [human.name for human in humans if human.name.endswith("e") == True]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = []
+c = [human.name for human in humans if human.name[0] == 'C' or human.name[0] == 'D' or human.name[0] == 'E' or human.name[0] == 'F' or human.name[0] == 'G']
 print(c)
 
 # Write a list comprehension that creates a list of all the ages plus 10.
 print("Ages plus 10:")
-d = []
+d = [human.age + 10 for human in humans]
 print(d)
 
 # Write a list comprehension that creates a list of strings which are the name
 # joined to the age with a hyphen, for example "David-31", for all humans.
 print("Name hyphen age:")
-e = []
+e = [ human.name + '-' + str(human.age) for human in humans]
```

Comment:
You could also do an f string here
```suggestion
e = [ f"{human.name}-{str(human.age)}" for human in humans]
```
then you don't need to concatenate the strings